File: flutter_ui/lib/external_page.dart
import 'package:flutter/material.dart';
import 'package:flutter_ui/my_scaffold.dart';

class ExternalPage extends StatelessWidget {
  const ExternalPage({super.key});

  @override
  Widget build(BuildContext context) {
    return const MyScaffold(
      title: 'External page',
      child: Center(
          child: Text(
        'External Page',
        style: TextStyle(
          fontWeight: FontWeight.bold,
          fontSize: 32,
          color: Colors.black,
          inherit: false,
        ),
      )),
    );
  }
}

File: rn_ui/pages/external_page.js
import React from "react";
import { SafeAreaView, Text } from "react-native";

const ExternalPage = () => {
    return (
        <SafeAreaView style={{ justifyContent: 'center', alignItems: 'center', flex: 1 }}>
            <Text style={{ fontSize: 24 }}>ExternalPage</Text>
        </SafeAreaView>
    );
}

export default ExternalPage;